bundle/edu.gemini.itc.shared/src/main/scala/edu/gemini/itc/shared/ItcService.scala

```diff
@@ -53,4 +59,10 @@ object ItcService {
 
   type Result = Validation[List[String], ItcResult]
 
+  /** Performs an ITC call on the given host. */
+  def calculate(kc: KeyChain, peer: Peer, source: SourceDefinition, obs: ObservationDetails, cond: ObservingConditions, tele: TelescopeDetails, ins: InstrumentDetails): Future[Result] =
+    TrpcClient(peer).withKeyChain(kc) future { r =>
+      r[ItcService].calculate(source, obs, cond, tele, ins)
+    }
```

Comment:
That's a good point. Indeed the ITC service is public, so I actually don't need any permissions here.